core/src/main/java/com/linecorp/armeria/server/VirtualHost.java

```diff
@@ -385,15 +385,18 @@ public Routed<ServiceConfig> findServiceConfig(RoutingContext routingCtx, boolea
         final Routed<ServiceConfig> routed = router.find(requireNonNull(routingCtx, "routingCtx"));
         switch (routed.routingResultType()) {
             case MATCHED:
+                maybeSetRoutingResult(routingCtx, routed);
```

Comment:
Oops, I found out that we need to update the Javadoc of this method because we don't return `Routed.empty()` anymore. Could you update it, please?

Reply:
So, if I understand this correctly, `Routed.empty()` could be returned only when `useFallbackService` is set to `false`?